library/core/src/fmt/float.rs

```diff
@@ -28,8 +28,8 @@ where
             *num,
             sign,
             precision,
-            buf.get_mut(),
-            parts.get_mut(),
+            buf.assume_init_mut(),
+            parts.assume_init_mut(),
```

Comment:
The FIXME above needs to also be adjusted.

Reply:
Also looks like the issue this is referencing got closed, and the new tracking issue does not even raise this question any more?

Reply:
Created https://github.com/rust-lang/rust/issues/76092